pkg/model/manifest_test.go

```diff
@@ -404,3 +404,148 @@ func TestSetManifestDefaultsFromDev(t *testing.T) {
 		})
 	}
 }
+
+func TestSetManifestBuildDefaults(t *testing.T) {
+	tests := []struct {
+		name             string
+		currentManifest  *Manifest
+		expectedManifest *Manifest
+	}{
+		{
+			name: "name filled, context empty, dockerfile empty",
```

Comment:
Could we add a test for when the 3 are filled?

Reply:
Sure! But if they are filled, the result is what you fill!
On it!!

Reply:
It was an interesting test @AgustinRamiroDiaz